ui/selectmenu.js

```diff
@@ -164,18 +167,12 @@ return $.widget( "ui.selectmenu", {
 						}
 					}
 					that.focusIndex = item.index;
-
```

Comment:
Same thing here, you've broken the context of working with the `focusIndex` property and working with the button.